tabbycat/breakqual/base.py

```diff
@@ -90,11 +90,23 @@ def _metric_name(metric):
                     name = annotator_class.name
                 return force_str(name)
 
+            required_metrics_message = ngettext(
+                "The break qualification rule %(rule)s requires the following "
+                "metric to be in the team standings precedence in order to "
+                "work: %(required)s;",
+                "The break qualification rule %(rule)s requires the following "
+                "metrics to be in the team standings precedence in order to "
+                "work: %(required)s;",
+                len(self.required_metrics),
+            )
+            missing_metrics_message = ngettext(
+                " and the following is missing: %(missing)s.",
+                " and the following are missing: %(missing)s.",
+                len(missing_metrics),
+            )
```

Comment:
I'd rather rephrase the message to only require one list than split it in two, as the ordering of clauses (or even agglutination) could determine how they're combined differently than in English. Maybe something like
> The %(rule)s break qualification rule is missing the following required metric in the team standings precedence: %(missing)s

@czlee ?

Reply:
Yeah, this kind of thing was a struggle for me in several places with internationalization. I agree, you can't assume that you can just concatenate sentences like that in other languages (though I can't actually name you a counterexample language off the top of my head for this case).

This error message evidently tried to be detailed enough to be as helpful as possible, but yeah, I agree that dropping the `required` list would be a minimal sacrifice in this context.

While I'm here, thanks so much for doing this @daankoning!!

Reply:
Implemented this!